render-tests/runtime-styling/set-style-layer-change-source-layer/style.json

```diff
@@ -0,0 +1,60 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "native": false,
+      "width": 64,
+      "height": 64,
+      "operations": [
+        [
+          "setStyle",
+          {
+            "version": 8,
+            "transition": {
+              "duration": 0
+            },
+            "sources": {
+              "vector": {
+                "type": "vector",
+                "tiles": [
+                  "local://tiles/{z}-{x}-{y}.mvt"
+                ]
+              }
+            },
+            "layers": [{
+              "id": "fill",
+              "type": "fill",
+              "source": "vector",
+              "source-layer": "building"
```

Comment:
Reason that `expected.png` looks right -- original style uses `source-layer: 'landuse'`.

Here's what the geojson dumps look like for landuse and building, respectively:

![screen shot 2016-11-16 at 6 46 45 pm](https://cloud.githubusercontent.com/assets/4974151/20374386/1e3b51e8-ac2d-11e6-9f04-ffd4fd93b44f.png)

![screen shot 2016-11-16 at 6 48 38 pm](https://cloud.githubusercontent.com/assets/4974151/20374414/5064c26c-ac2d-11e6-8f0a-b0db53965f6e.png)

(the rendered image is right around the traffic circle)